paymentsheet/src/main/java/com/stripe/android/paymentsheet/navigation/PaymentSheetScreen.kt

```diff
@@ -219,6 +228,7 @@ internal sealed interface PaymentSheetScreen {
         override val topContentPadding: Dp = 0.dp
         override val bottomContentPadding: Dp = horizontalModeBottomContentPadding
         override val walletsDividerSpacing: Dp = horizontalModeWalletsDividerSpacing
+        override val animationStyle: AnimationStyle = AnimationStyle.FullPage
```

Comment:
Why does this screen not have `PrimaryButtonAnchored` as its animation style? I would expect it to have the same animation as `AddAnotherPaymentMethod`

Reply:
The anchored behavior is primarily between Select -> Add another.

Add first is always first, so it doesn't need the primary button anchored when animating to it.

Reply:
Animations are only chosen on the target screen. So navigating from any screen to the edit card brand screen will result in a full page transition. I can grab some screen recordings of before/after.

Reply:
Before Card updating
https://github.com/user-attachments/assets/dadc1e27-2b37-463f-9780-33907ff2193f

After card updating
https://github.com/user-attachments/assets/3aefb646-978a-41ad-b7ef-43da5f926c1b

Reply:
nice, thank you!